eZ/Publish/Core/FieldType/Keyword/KeywordStorage.php

```diff
@@ -60,7 +60,7 @@ public function deleteFieldData(VersionInfo $versionInfo, array $fieldIds, array
     {
         $gateway = $this->getGateway($context);
         foreach ($fieldIds as $fieldId) {
-            $gateway->deleteFieldData($fieldId);
+            $gateway->deleteFieldData($fieldId, $versionInfo);
```

Comment:
Right now it will be called per every `$fieldId` yet you already know here that you should not delete field data.